.travis.yml

```diff
@@ -11,10 +11,10 @@ env:
         # New FreeType causes changes in text output.
         - NAME="Latest everything."
           PYTHON_VERSION=3.6
-          PACKAGES="numpy matplotlib freetype<2.8 scipy"
+          PACKAGES="numpy matplotlib scipy"
```

Comment:
How come no freetype?

Reply:
The Matplotlib from the testing label uses a fixed copy of FreeType bundled into Matplotlib, so it will never change (at least until Matplotlib rebuilds all its images with a new version, which is unlikely unless some new comparison method is used.)